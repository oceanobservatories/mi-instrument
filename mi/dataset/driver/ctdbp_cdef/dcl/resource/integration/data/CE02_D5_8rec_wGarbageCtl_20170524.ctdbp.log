2017/05/24 05:00:04.384 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/24 05:00:07.806 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/24 05:02:01.289 # 10.5122,  3.60285,    6.543, 24 May 2017 05:01:54
2017/05/24 05:03:07.793 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/24 05:15:07.483 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/24 05:17:50.288 # 10.5086,  3.60228,    6.369, 24 May 2017 05:17:43
2017/05/24 05:18:06.767 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/24 05:32:10.314 [ctdbp:DLOGP3]:Qú*ë
2017/05/24 05:47:10.738 [ctdbp:DLOGP3]:Qú*ë
2017/05/24 06:02:10.462 [ctdbp:DLOGP3]:Qú*ë
2017/05/24 06:15:04.899 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/24 06:17:10.186 [ctdbp:DLOGP3]:Qú*ë
2017/05/24 06:30:10.323 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/24 06:32:03.290 # 10.4627,  3.60109,    6.229, 24 May 2017 06:31:56
2017/05/24 06:32:13.290 # 10.4589,  3.60092,    6.841, 24 May 2017 06:32:06
2017/05/24 06:33:09.776 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/24 06:47:09.684 [ctdbp:DLOGP3]:Qú*ë
2017/05/24 07:00:04.112 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/24 07:02:09.408 [ctdbp:DLOGP3]:Qú*ë
2017/05/24 07:15:09.829 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/24 07:16:43.290 # 10.4193,  3.59904,    6.630, 24 May 2017 07:16:36
2017/05/24 07:17:13.303 # 10.4224,  3.59937,    6.507, 24 May 2017 07:17:06
2017/05/24 07:18:09.779 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/24 07:30:09.170 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/24 07:31:12.290 # 10.4436,  3.59998,    6.158, 24 May 2017 07:31:05
2017/05/24 07:32:12.291 # 10.4255,  3.59910,    7.125, 24 May 2017 07:32:05
2017/05/24 07:33:08.772 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
