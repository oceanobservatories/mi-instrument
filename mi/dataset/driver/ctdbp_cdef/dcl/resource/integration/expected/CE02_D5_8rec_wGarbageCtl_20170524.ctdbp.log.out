
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704590921.2890000343 |          5 |         3704590914 | 3704590921.2890000343 |      port_timestamp |           ok |      3.60285 |             null |                     null |    6.543 | 10.5122
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704591870.2880001068 |          5 |         3704591863 | 3704591870.2880001068 |      port_timestamp |           ok |      3.60228 |             null |                     null |    6.369 | 10.5086
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704596323.2899999619 |          5 |         3704596316 | 3704596323.2899999619 |      port_timestamp |           ok |      3.60109 |             null |                     null |    6.229 | 10.4627
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704596333.2899999619 |          5 |         3704596326 | 3704596333.2899999619 |      port_timestamp |           ok |      3.60092 |             null |                     null |    6.841 | 10.4589
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704599003.2899999619 |          5 |         3704598996 | 3704599003.2899999619 |      port_timestamp |           ok |      3.59904 |             null |                     null |     6.63 | 10.4193
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704599033.3029999733 |          5 |         3704599026 | 3704599033.3029999733 |      port_timestamp |           ok |      3.59937 |             null |                     null |    6.507 | 10.4224
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704599872.2899999619 |          5 |         3704599865 | 3704599872.2899999619 |      port_timestamp |           ok |      3.59998 |             null |                     null |    6.158 | 10.4436
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704572800 | telemetered | 3704599932.2909998894 |          5 |         3704599925 | 3704599932.2909998894 |      port_timestamp |           ok |       3.5991 |             null |                     null |    7.125 | 10.4255

(8 rows)
